src/FormBuilder/resources/js/form-builder/src/supports/field-settings/slots.ts

```diff
@@ -0,0 +1,18 @@
+import {createSlotFill} from '@wordpress/components';
+
+/**
+ * Slots for used within the field settings inspector controls. This allows plugins and such to add further controls to
+ * the inspector sections.
+ *
+ * @unreleased
+ */
+const {Slot: FieldSettingsSlot, Fill: FieldSettingsFill} = createSlotFill('GiveWP/FieldSettings/FieldSettingSlot');
+const {Slot: DisplaySettingsSlot, Fill: DisplaySettingsFill} = createSlotFill(
+    'GiveWP/FieldSettings/DisplaySettingSlot'
+);
+
+window.givewp.slots = window.givewp.slots || {};
+window.givewp.slots.FieldSettingsFill = FieldSettingsFill;
+window.givewp.slots.DisplaySettingsFill = DisplaySettingsFill;
```

Comment:
I think we can remove this since your already assigning in registerHooks() right?

Reply:
Oops! Good catch. Resolved in 460cbc8